src/styles/main.scss

```diff
@@ -1,3 +1,94 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+@import "./utils/reset";
+
+.calender {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: $calender-padding;
+  gap: $cell-gap;
+  width: ($cell-width * 7) + ($cell-gap * 6) + ($calender-padding * 2);
+
+  &--position {
+    position: absolute;
+    top: $calender-positioning;
+    left: $calender-positioning;
+    transform: translate(-$calender-positioning, -$calender-positioning);
+  }
+
+  &__cell {
+    position: relative;
+    height: 100px;
+    width: $cell-width;
+    line-height: 100px;
+    text-align: center;
+    border: 1px solid #000;
+    background-color: #eee;
+    font-size: 30px;
+
+    @for $count-days from 1 through 31 {
+      &:nth-child(#{$count-days})::before {
+        content: "#{$count-days}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      animation: round 0.5s, color 0.5s;
```

Comment:
it's too much on hover)

Reply:
is not fixed. Just change your color transition to 250ms